bert/src/bert_padding.py

```diff
@@ -0,0 +1,138 @@
+
```

Comment:
This is honestly pretty confusing. More documentation would be helpful at some point. Not a blocker.

Reply:
I think we'll leave this and the flash attention documentation for now, since it's pretty low-level code that the user will hopefully not need to understand. If that's leading to confusion, though, we'll revisit.